pxemgr/ipxe_handlers.go

```diff
@@ -36,7 +36,12 @@ func (mgr *pxeManagerT) ipxeBootScript(w http.ResponseWriter, r *http.Request) {
 
 	buffer := bytes.NewBufferString("")
 	buffer.WriteString("#!ipxe\n")
-	buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
+	if mgr.config.Network.UEFI {
+		buffer.WriteString(fmt.Sprintf("kernel %s/images/vmlinuz coreos.autologin maybe-install-coreos=stable console=ttyS0,115200n8 mayu=%s next-script=%s\n initrd=initrd.cpio.gz", mgr.pxeURL(), mgr.pxeURL(), mgr.pxeURL()+"/first-stage-script/__SERIAL__"))
```

Comment:
1. its extra argument that needs to be add to kernel in order to boot with UEFI it comes from here - https://github.com/coreos/bugs/issues/1239#issuecomment-268000634
2. no, it should be only`initrd=initrd.cpio.gz`